backend/api_v2/api_deployment_views.py

```diff
@@ -73,21 +74,26 @@ def post(
     def get(
         self, request: Request, org_name: str, api_name: str, api: APIDeployment
     ) -> Response:
-        execution_id = request.query_params.get("execution_id")
-        include_metadata = (
-            request.query_params.get(ApiExecution.INCLUDE_METADATA, "false").lower()
-            == "true"
-        )
-        if not execution_id:
-            raise InvalidAPIRequest("execution_id shouldn't be empty")
-        response: ExecutionResponse = DeploymentHelper.get_execution_status(
-            execution_id=execution_id
-        )
+        serializer = ExecutionQuerySerializer(data=request.query_params)
+        serializer.is_valid(raise_exception=True)
+
+        execution_id = serializer.validated_data.get(ApiExecution.EXECUTION_ID)
+        include_metadata = serializer.validated_data.get(ApiExecution.INCLUDE_METADATA)
```

Comment:
@Deepak-Kesavan why haven't we added `include_metadata` in our [docs](https://docs.unstract.com/unstract/unstract_platform/api_deployment/unstract_api_deployment_status_api/) or in the Postman collection we export?

Reply:
@chandrasekharan-zipstack Not sure about the docs. But i remember adding it in postman collection. Need to check if that is missed out in v2 changes.

Reply:
@pk-zipstack can you help raise a PR in the unstract-docs repo to add `include_metadata` parameter [here](https://docs.unstract.com/unstract/unstract_platform/api_deployment/unstract_api_deployment_status_api/#parameters)?